src/js/index/todolist.js

```diff
@@ -0,0 +1,121 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $todoList = document.querySelector("#todo-list");
+const $todoCount = document.querySelector("strong");
+const $filters = document.querySelector(".filters");
+
+$todoInput.addEventListener("keyup", onAddTodoItem);
+$todoList.addEventListener("click", onToggleTodoItem);
+$todoList.addEventListener("dblclick", onEditTodoItem);
+$filters.addEventListener("click", onFilter);
+
+function onAddTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
+    if (event.key === "Enter" && todoTitle !== "") {
+        todoList.insertAdjacentHTML("beforeend", renderTodoItemTemplate(todoTitle));
+        updateTodoCount();
+        event.target.value = "";
+        changeState();
+    }
+}
+
+function onToggleTodoItem(event) {
+    if (event.target.classList.contains("toggle")) {
+        event.target.closest("li").classList.toggle("completed");
+        event.target.closest("input").toggleAttribute("checked");
+        changeState()
+    } else if (event.target.classList.contains("destroy")) {
+        event.target.closest("li").remove();
+        updateTodoCount();
+        changeState();
+    }
+}
+
+function onEditTodoItem(event) {
+    if (event.target.classList.contains("label")) {
+        const originalValue = event.target.innerText;
+        event.target.closest("li").classList.add("editing");
+        event.target.parentElement.parentElement.querySelector(".edit").focus();
+        event.target.closest("li")
+            .addEventListener("keyup", function (event) {editTodoItem(event, originalValue)});
```

Comment:
```suggestion
            .addEventListener("keyup", (event) => editTodoItem(event, originalValue));
```

인라인 함수는 arrow function 을 쓰는게 일반적입니다~~ 

https://ko.javascript.info/arrow-functions-basics